crates/edr_instrument/src/coverage.rs

```diff
@@ -97,7 +98,7 @@ pub fn instrument_code(
         }
     }
 
-    instrumented_source.push_str("\nimport \"hardhat/coverage.sol\";");
+    instrumented_source.push_str(&format!("\nimport \"{coverage_library_path}\";"));
```

Comment:
Note that this could result in invalid syntax, for example if the library path contains a quote symbol.

I think we should try to avoid that at this point, rather than failing to compile the file at a later point, by either validating the library path or escaping invalid characters (quotes and backslashes?).

Reply:
Good call. Addressed in https://github.com/NomicFoundation/edr/pull/904/commits/da1e5c40fc64da7bf9ddd126450525ea5950ee0e

Reply:
Very cool solution!